### PR TITLE
feat(construct): fixed bugs in image generation construct 

### PR DESCRIPTION
**Pull request by dineshSajwan**
Fixes #
1. updated image generation construct with comprehend and s3 actions 
2. updated  image generation construct with GraphQL api type
3. Updated template description format
---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the project license.

### CODE REVIEW

**Review by krokoko — DISMISSED** [review]

**Review by krokoko — DISMISSED** [review]

File: apidocs/classes/ContentGenerationAppSyncLambda.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ ___
 
 ### graphqlApi
 
-• `Readonly` **graphqlApi**: `IGraphqlApi`
+• `Readonly` **graphqlApi**: `GraphqlApi`
 
 Returns an instance of appsync.IGraphqlApi created by the construct
```
Comment (scottschreckengaust) [inline apidocs/classes/ContentGenerationAppSyncLambda.md:145]:
```suggestion
Returns an instance of appsync.GraphqlApi created by the construct
```

File: src/patterns/gen-ai/aws-contentgen-appsync-lambda/index.ts [hunk 2]
```diff
@@ -154,7 +154,7 @@ export class ContentGenerationAppSyncLambda extends BaseClass {
   /**
    * Returns an instance of appsync.IGraphqlApi created by the construct
    */
-  public readonly graphqlApi: appsync.IGraphqlApi;
+  public readonly graphqlApi: appsync.GraphqlApi;
```
Comment (scottschreckengaust) [inline src/patterns/gen-ai/aws-contentgen-appsync-lambda/index.ts:157]: same question as above `IGraphqlApi`?

File: src/patterns/gen-ai/aws-contentgen-appsync-lambda/index.ts [hunk 3]
```diff
@@ -154,7 +154,7 @@ export class ContentGenerationAppSyncLambda extends BaseClass {
   /**
    * Returns an instance of appsync.IGraphqlApi created by the construct
```
Comment (scottschreckengaust) [inline src/patterns/gen-ai/aws-contentgen-appsync-lambda/index.ts:155]:
```suggestion
   * Returns an instance of appsync.GraphqlApi created by the construct
```

File: apidocs/classes/ContentGenerationAppSyncLambda.md (hunk 1 above)
Comment (dineshSajwan) [reply apidocs/classes/ContentGenerationAppSyncLambda.md:145]: Yeah I changed IGraphqlApi to GraphqlApi to support graphqlurl property in CFN output.

File: src/patterns/gen-ai/aws-contentgen-appsync-lambda/index.ts (hunk 2 above)
Comment (dineshSajwan) [reply src/patterns/gen-ai/aws-contentgen-appsync-lambda/index.ts:157]: GraphqlApi  is required.

File: src/patterns/gen-ai/aws-contentgen-appsync-lambda/index.ts [hunk 4]
```diff
@@ -154,7 +154,7 @@ export class ContentGenerationAppSyncLambda extends BaseClass {
...
    */
-  public readonly graphqlApi: appsync.IGraphqlApi;
+  public readonly graphqlApi: appsync.GraphqlApi;
   /**
    * Returns an instance of appsync.IGraphqlApi created by the construct
```
Comment (scottschreckengaust) [inline src/patterns/gen-ai/aws-contentgen-appsync-lambda/index.ts:159]:
```suggestion
   * Returns an instance of appsync.GraphqlApi created by the construct
```

**Review by scottschreckengaust — APPROVED** [review]

**Review by krokoko — APPROVED** [review]

**Pull request merged**